09-threads/lab/solution/src/bg/sofia/uni/fmi/mjt/restaurant/Chef.java

```diff
@@ -0,0 +1,45 @@
+package bg.sofia.uni.fmi.mjt.restaurant;
+
+public class Chef extends Thread {
+
+	private final int id;

```

Comment:
Please fix indentation of all classes to 4 spaces.

Reply:
Done.